cnf/ext/settings.bnd

```diff
@@ -0,0 +1,16 @@
+-releaserepo: Release
```

Comment:
What is the value of moving these from build.bnd? You just leave an empty build.bnd behind when these values should still be there. Users can add there own ext files to override the values in the central build.bnd file.

Reply:
The idea is to keep the build.bnd empty in our template, to avoid conflicts when reapplying the workspace template to update to e.g. a newer bnd version. A build.bnd will always be there and chances are good, that people will edit it. If so, one can simply uncheck it and it will not be overwritten on update. With the current version of the workspace, I could always update our projects, by simply ignoring the osgi_repo.bnd plus its maven file and the settings.bnd.

Reply:
So you just move the problem to this file since the template will want to replace this file too. There is no practical difference that I can see.

Reply:
Having the default values in build.bnd is nice since an ext bnd file can override it since they are read _after_. If you put the defaults in an ext bnd file, then someone must edit that file (or create another ext bnd file which is read after the settings.bnd fie) to override.

Reply:
I would say we both have a valid point here. I would suggest we discuss this on our Friday bash and wait what the others think.